lib/routes/account.js

```diff
@@ -477,7 +478,8 @@ module.exports = function (
                     uid: emailRecord.uid,
                     kA: emailRecord.kA,
                     wrapKb: wrapKb,
-                    emailVerified: emailRecord.emailVerified
+                    emailVerified: emailRecord.emailVerified,
+                    tokenVerificationId: doSigninConfirmation ? tokenVerificationId : undefined
```

Comment:
In addition to these two changes where we *do* set a `tokenVerificationId`, I think it would be worthwhile to update the password-change-related endpoints with a comment to explain why we're *deliberately not* setting one.  If we don't get back to revisiting those for a while, our future selves will thank us.